include/build_config/SDL_build_config_windows.h

```diff
@@ -95,7 +95,7 @@ typedef unsigned int uintptr_t;
 #if defined(_WIN32_MAXVER) && _WIN32_MAXVER >= 0x0603  /* Windows 8.1 SDK */
 #define HAVE_SHELLSCALINGAPI_H 1
 #endif
-#if defined(_WIN32_MAXVER) && _WIN32_MAXVER >= 0x0601  /* Windows 7 SDK */
+#if defined(_MSC_VER) && defined(_WIN32_MAXVER) && _WIN32_MAXVER >= 0x0601  /* Windows 7 SDK */
 #define HAVE_SHOBJIDL_CORE_H 1
 #endif
```

Comment:
This whole block should be removed because we're not checking that define anymore, right?

Reply:
Yes, that seems fine.

Reply:
We don’t need to add complexity here 😊